src/topology/basic.lean

```diff
@@ -434,14 +434,15 @@ localized "notation `𝓝` := nhds" in topological_space
 
 lemma nhds_def (a : α) : 𝓝 a = (⨅ s ∈ {s : set α | a ∈ s ∧ is_open s}, 𝓟 s) := rfl
 
-/-- The open sets containing `a` are a basis for the neighborhood filter. -/
+/-- The open sets containing `a` are a basis for the neighborhood filter. See `nhds_basis_opens'`
+for a variant using open neighborhoods instead. -/
 lemma nhds_basis_opens (a : α) : (𝓝 a).has_basis (λ s : set α, a ∈ s ∧ is_open s) (λ x, x) :=
 has_basis_binfi_principal
   (λ s ⟨has, hs⟩ t ⟨hat, ht⟩, ⟨s ∩ t, ⟨⟨has, hat⟩, is_open_inter hs ht⟩,
     ⟨inter_subset_left _ _, inter_subset_right _ _⟩⟩)
   ⟨univ, ⟨mem_univ a, is_open_univ⟩⟩
 
-/-- A filter includes the neighborhoods of `a` iff it includes the open sets containing `a`. -/
+/-- A filter has the neighborhoods of `a` iff it has the open sets containing `a`. -/
```

Comment:
I don't understand this use of `has`. I could never understand this docstring without reading the statement of the lemma. I would simply remove it.

Reply:
```suggestion
/-- A filter lies below the neighborhood filter at `a` iff it contains every open set around `a`. -/
```

Reply:
I agree that changing to `has` made the text more confusing. Mario's suggestion looks more clear to me.